CHANGELOG.rst

```diff
@@ -1,116 +1,17 @@
 ==============
-foursight-cgap
+foursight-smaht
 ==============
 
 
 ----------
 Change Log
 ----------
 
-3.6.0
+0.0.1
```

Comment:
Maybe mention where this is forked from? Possibly even a URL to the particular commit point of fork?